### ISSUE TITLE
backend

### ISSUE THREAD

**Issue by mohamed29ac** [experiencor/keras-yolo3#39]
Could you explain how you got the pretrained weights for backend(backend.h5)

**Comment by Jasonsun1993** [comment]
Just download it from https://1drv.ms/u/s!ApLdDEW3ut5fgQXa7GzSlG-mdza6

**Comment by Jasonsun1993** [comment]
@mohamed29ac How many epochs did you train? And how much is the mAP ?

**Comment by mohamed29ac** [comment]
@Jasonsun1993 for training the raccoon I used 100 epochs and got 96 mAP. But When I tried in my own dataset to detect a vehicle the result isn't good.
![image](https://user-images.githubusercontent.com/21984270/39668856-9fc87e0e-510c-11e8-81f9-4df7e3e39eb4.png)

**Comment by Jasonsun1993** [comment]
@mohamed29ac It seems that the training is on process from the figure you provided. It is OK to use the same config.json file to train your own dataset (but I suggest you to adjust the input_size, learning rate and anchors). In the beginning of training, the loss will be very high. With the increase of iteration, loss will go down. You need just to wait to see the result mAP. Additionally, if you had trained enough epochs on your own dataset, but still got bad results. Try to expand your dataset because the model may be overfitted.

**Comment by casfranco** [comment]
backend weights are no longe availables -- This item might not exist or is no longer available
